web/src/test/java/io/fabric8/launcher/web/endpoints/inputs/LaunchProjectileInputTest.java

```diff
@@ -0,0 +1,94 @@
+package io.fabric8.launcher.web.endpoints.inputs;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.json.JSONObject;
```

Comment:
Can we use Jackson here instead of org.json?

Reply:
We can. Any reason to use a different library?

Reply:
Because Jackson is used everywhere in the code, but I was wondering if Mockito would be a better choice instead